test/public/overview.test.js

```diff
@@ -17,18 +17,18 @@
  */
 
 const assert = require('assert');
-const puppeteer = require('puppeteer');
 const pti = require('puppeteer-to-istanbul');
+const puppeteer = require('puppeteer');
 const { server } = require('../../lib/application');
 
 module.exports = function () {
     // Configure this suite to have a default timeout of 10s
-    this.timeout(10000);
+    this.timeout(5000);
 
     let page;
     let browser;
     let PORT;
```

Comment:
This can either be a local variable or use string interpolation.

Reply:
What is the added value of string interpolation, or putting it in a single variable since it is only used once...